deployments/polygon/usdc/migrations/1689168483_add_maticx_collateral.ts

```diff
@@ -98,6 +96,10 @@ export default migration("1689168483_add_maticx_collateral", {
     trace(`Created proposal ${proposalId}.`);
   },
 
+  async enacted(deploymentManager: DeploymentManager): Promise<boolean> {
+    return true;
+  },
+
```

Comment:
This shouldn't be true unless the proposal was actually created.

Reply:
cool will remove